assets/constants.js

```diff
@@ -498,6 +498,8 @@ var constants = {
     ]
   }],
 
+  "INTERACTION_IDS_WITHOUT_ANSWER_DETAILS": ["EndExploration", "Continue"],
```

Comment:
You can move the feconf constant in constant.js and that value can be used both in frontend and backend. (Instead of creating two variable.)

Reply:
Also, make sure to add a comment on the top of this constant.

Reply:
https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/app.constants.ts#L19

Reply:
Done